src/modules/src/commander.c

```diff
@@ -174,10 +175,6 @@ static void commanderCrtpCB(CRTPPacket* pk)
   crtpCache.targetVal[!crtpCache.activeSide] = *((struct CommanderCrtpValues*)pk->data);
   crtpCache.activeSide = !crtpCache.activeSide;
   crtpCache.timestamp = xTaskGetTickCount();
-
-  if (crtpCache.targetVal[crtpCache.activeSide].thrust == 0) {

```

Comment:
I'd prefer to keep the thrust lock -- it's a good safety feature, especially when using transmitters with a non-spring loaded throttle that might not be at zero on power-on, and with BigQuad builds.